lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
+        scope = scope || 'global';
+        return new Promise(function(resolve, reject) {
```

Comment:
Chatted on slack, appeared related to the fact that Bluebird doesn't coerce an array of promises by default without prompting via `.map`, `.all` or `.spread`.